site/content/docs/manifest/backend-service.md

```diff
@@ -151,5 +151,10 @@ Key-value pairs that represent secret values from [AWS Systems Manager Parameter
 
 <div class="separator"></div>
 
+<a id="storage" href="#storage" class="field">`storage`</a> <span class="type">Map</span>  
+The Storage section lets you specify external EFS volumes for your containers and sidecars to mount. This allows you to access persistent storage across regions for data processing or CMS workloads. For more detail, see the [storage](../developing/storage.md) page.
```

Comment:
nice, I think we should still include the subfields here though. These pages are a nice place for customers to see all the available fields in the manifest, on the other hand we don't need to go into as much detail.